src/ocamlorg_web/lib/handler.ml

```diff
@@ -246,7 +246,11 @@ let blog req =
          Data.Planet.Post.all)
   in
   let featured = Data.Planet.featured_posts |> List.take 3 in
-  let news = Data.News.all |> List.take 20 in
+  let number_of_news =
+    List.length featured + List.length current_items
+    |> float_of_int |> ( *. ) 1.2 |> int_of_float
```

Comment:
```suggestion
    |> float_of_int |> ( *. ) 1.3 |> int_of_float
```

Looks to me like we can risk a few more items here.

Indeed, it will never be exactly the same height, and this improves things for now, even if in very rare cases the news may be taller.